scripts/maintenance/migrate_project/Makefile

```diff
@@ -0,0 +1,22 @@
+.DEFAULT_GOAL := build
+
+SHELL := /bin/bash
+
+IMAGE_NAME := migrate-utils
+TAG := latest
+
+build:
+	# creating python virtual environment
+	docker-compose -f docker-compose-build.yaml build
```

Comment:
why using docker-compose to build and docker CLI to run? Either one or the other right? This just adds an extra compose file for build ... I would rather use
```docker build   -t migrate-utils $(REPO_DIR)```

e.g. see how we deduce REPODIR in our Makefile

Reply:
The only reason I had to add a file, is because I needed to inject the context pointing outside this directory. From what I understand it is only possible to do it with docker-compose

Reply:
Even after implementing the `REPODIR`, it still does not allow me to build. Will have to keep the `docker-compose-build.yaml`

Reply:
@GitHK you can also do it for any folder, look at the other scripts in the /scripts folder.
the context when you build is defined where you run the ```docker``` command.

Reply:
🎉 finally managed to fix this build context. Such a pin!